src/ansys/workbench/core/public_api.py

```diff
@@ -254,7 +259,13 @@ def exit(self):
 
 
 def launch_workbench(
-    release="242", client_workdir=None, server_workdir=None, host=None, username=None, password=None
+    no_ui=False,
```

Comment:
shouldn't we place this new parameter at the end for compatibility with calls that do not use named parameters?

Reply:
Positional parameters are NOT recommended. And all of the parameters are actually optional.